app/models/pair_request.rb

```diff
@@ -50,4 +49,23 @@ class PairRequest < ApplicationRecord
   def started?
     self.when <= Time.current
   end
+
+  def pending_feedback?(current_user)
+    feedback = references.find_by(author: current_user, status: 'draft')
```

Comment:
this can be simplified to just `references.find_by(author: current_user).draft?`

Reply:
Ending on `.draft?` will make it return `true/false`. It could be `references.draft.find_by(author: current_user)` though!

Reply:
shouldn't this be returning true/false??

Reply:
Guys, hate to say this, but this method is not being used. This is from before I added the helper 😁

Reply:
but this method  should return the id of the draft Feedback associated to the user if it exists, nil when it doesn't.